lisa/sut_orchestrator/azure/common.py

```diff
@@ -94,6 +94,7 @@ class AzureNodeSchema:
         default=None, metadata=field_metadata(data_key="shared_gallery")
     )
     vhd: str = ""
+    hyperv_generation: str = "V1"
```

Comment:
The type should be an integer, and limited to 1, or 2. It can reduce human error on how to assign this value. And it should be converted to "V1" or "V2" in code.

Reply:
Is it possible to know from the marketplace image, it's a v1 or v2?